src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
+
+  @include media($nav-width) {
+    border-bottom: 1px solid $color-gray-light;
+  }
+
+  a {
+    border-bottom: none;
+  }
+
+  em {
```

Comment:
I'm going to add more info in our documentation: https://docs.google.com/document/d/1zMNdp5UqJlMUWmDOboirop2kBMg5U50cOfSOajA_Vt8/edit?disco=ou2agy5g7l5u